volumes/miovision/sql/function-find_gaps.sql

```diff
@@ -14,10 +14,36 @@ DECLARE tot_gaps integer;
 BEGIN
 
 -- FIRST, FIND ALL GAPS
```

Comment:
Is this comment still in the right spot?

Reply:
Ahh, no, will move it.